Modules/Quotas/src/Quotas.ConsumerApi/Controllers/QuotasController.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Backbone.BuildingBlocks.API;
+using Backbone.BuildingBlocks.API.Mvc;
+using Backbone.BuildingBlocks.API.Mvc.ControllerAttributes;
+using Backbone.Modules.Quotas.Application.Identities.Queries.ListQuotasForIdentity;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Backbone.Modules.Quotas.ConsumerApi.Controllers;
+
+[Route("api/v1/[controller]")]
+[Authorize("OpenIddict.Validation.AspNetCore")]
+public class QuotasController : ApiControllerBase
+{
+    public QuotasController(IMediator mediator) : base(mediator) { }
+
+    [HttpGet]
+    [ProducesResponseType(typeof(HttpResponseEnvelopeResult<IEnumerable<QuotaGroupDTO>>), StatusCodes.Status200OK)]
```

Comment:
This is a lie. What's returned is a `ListQuotasForIdentityResponse`...

Reply:
Ok, true. Now I've changed it like so (as part of the request just above this one):
```csharp
        return Ok(response.Items);
```

The type of `Items` is `IEnumerable<QuotaGroupDTO>`. Should I then leave it as it is now?